test-cases/custom/longevity-gce-custom-d1-posts.yaml

```diff
@@ -0,0 +1,28 @@
+
+test_duration: 900
+n_db_nodes: 3
+n_test_oracle_db_nodes: 1
```

Comment:
What is the oracle size/type?

I know the default for AWS, what is it for GCE ?

Reply:
I took a look, we are running without oracle, we don't have a setup for it in GCE.

@roydahan I think we'll leave this effort out for now, we'll can circle back to create the oracle setup, but without oracle I don't see much point in running gemini, it doesn't add much value.